src/ChatGPTBrowserClient.js

```diff
@@ -178,6 +178,71 @@ export default class ChatGPTBrowserClient {
         return response;
     }
 
+    async deleteConversation(conversationID) {
+        const url = this.options.reverseProxyUrl || 'https://chat.openai.com/backend-api/conversation';
+        const opts = {
+            method: 'PATCH',
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: `Bearer ${this.accessToken}`,
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36',
+                Cookie: this.cookies || undefined,
+            },
+
+            body: JSON.stringify({
+                is_visible: false,
+            }),
+        };
+        const { debug } = this.options;
+
+        // eslint-disable-next-line no-async-promise-executor
+        const response = await new Promise(async (resolve, reject) => {
```

Comment:
IMO you can just return `new Promise...` as people can `await deleteConversation` anyway.